src/components/Entry/Entry.js

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react'
+import {useNavigate} from 'react-router-dom'
+import { Rating } from 'react-simple-star-rating'
+import'./Entry.css'
+
+function Entry ({setCurrentUser, id, title, artists, date, notes, image, rating, masterId}) {
```

Comment:
Just a suggestion, but I think this should be restructured in the component instead of passed in as a bunch of props.